javascript/javascript1/week1/house.js

```diff
@@ -0,0 +1,34 @@
+// A house price estimator
+
+  // for Peter
+  let PeterVolumeInMeters=10*10*8;
+  let PeterGardenSizeInM2=100;
+  const peterCost=2500000;
+  const PeterHousePrice=PeterVolumeInMeters*2.5*1000+PeterGardenSizeInM2*300;
+  console.log("Peter calculated it will cost "+PeterHousePrice);
+  if(PeterHousePrice<peterCost){
+      console.log("Peter will pay too much for a house");
+  }
+  else
+  if(PeterHousePrice>peterCost){
+      console.log("Peter will pay too little for a house");
+  }
+  else{
+      console.log("Peter will pay is a good price for a house");
+  }
+   // For Julyie
```

Comment:
Julyie seems to be a typo; the correct name is "Julia" as per the original problem statement right...